doc/integrations/chdb.md

```diff
@@ -0,0 +1,115 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.7
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Use chDB from Jupyter using JupySQL
+    keywords: jupyter, sql, jupysql, chDB
+    property=og:locale: en_US
+---
+
+# chDB
+
+```{note}
+JupySQL also supports chDB with dbapi connection (no SQLAlchemy needed)
+```
+
+JupySQL integrates with chDB so you can run SQL queries in a Jupyter notebook. Jump into any section to learn more!
+
++++
+
+## Pre-requisites for `.parquet` file
+
+```{code-cell} ipython3
+%pip install jupysql chdb pyarrow --quiet
+```
+
+```{code-cell} ipython3
+from chdb import dbapi
+conn = dbapi.connect()
+
+%load_ext sql
+%sql conn --alias chdb
+```
+
+### Get a sample `.parquet` file:
+
++++
+
+```{code-cell} ipython3
+%%bash
+wget 'https://datasets.clickhouse.com/hits_compatible/athena_partitioned/hits_0.parquet' -q -O hits_0.parquet
+```
+
+### Query on S3/HTTP/File
+
++++
+
+
+Query chDB on local file
+
+```{code-cell} ipython3
+%%sql
+SELECT
+    RegionID, SUM(AdvEngineID), COUNT(*) AS c, AVG(ResolutionWidth), COUNT(DISTINCT UserID)
+FROM file('hits_0.parquet')
+GROUP BY
+    RegionID
+ORDER BY c
+DESC LIMIT 10
+```
+
+Run chDB on HTTP (Parquet, CSV, JSON ...)
+```{code-cell} ipython3
+%%sql
+SELECT
+    RegionID, SUM(AdvEngineID), COUNT(*) AS c, AVG(ResolutionWidth), COUNT(DISTINCT UserID)
+FROM url('https://datasets.clickhouse.com/hits_compatible/athena_partitioned/hits_0.parquet')
+-- query on s3 --
+--  FROM s3('xxxx')
+GROUP BY
+    RegionID
+ORDER BY c
+DESC LIMIT 10
+```
+
+## Pre-requisites for another ClickHouse server
+
+```{code-cell} ipython3
+%pip install jupysql chdb --quiet
+```
+
+```{code-cell} ipython3
+from chdb import dbapi
+conn = dbapi.connect()
+
+%load_ext sql
+%sql conn --alias chdb
+```
```

Comment:
looks like this section is showing how to add a new connection, I think we can delete it. we already have documentation on how to manage connections